mock-core-configs/etc/mock/alma-8-x86_64.cfg

```diff
@@ -0,0 +1,8 @@
+include('/etc/mock/templates/alma-8.tpl')
+include('/etc/mock/templates/epel-8.tpl')
+
+config_opts['target_arch'] = 'x86_64'
+config_opts['legal_host_arches'] = ('x86_64',)
+config_opts['hostname'] = 'mock'
+config_opts['environment']['HOSTNAME'] = 'mock'
```

Comment:
Why these two lines about `hostname`? I guess this should go away unless you have some strong reason.